index.js

```diff
@@ -47,6 +47,16 @@ function Converter (sm, opts) {
   }
 }
 
+function findLastSourceMapComment(comment, line){
+  if (line.indexOf('sourceMappingURL=data:') > -1) comment = line;
+  return comment;
```

Comment:
Actually just realizing that this is inefficient.
You should just walk backwards through the lines via a simple for loop and `break` right when you find the comment.

Otherwise you'll keep visiting all lines even after you already found the comment.
Also you're starting at the first line although you know it's most likely the last.

We can keep working on this in the `fromsource` branch.
Just keep adding commits on top, we can squash right before merging into master.